client/utils.go

```diff
@@ -35,3 +41,33 @@ func Paginate(numObjs, page, limit, defLimit int) (start, end int) {
 
 	return start, end
 }
+
+// ReadPageRequest reads and builds the necessary page request flags for pagination.
+func ReadPageRequest(flagSet *pflag.FlagSet) (*query.PageRequest, error) {
+	pageKey, err := flagSet.GetString(flags.FlagPageKey)
+	if err != nil {
+		return nil, err
```

Comment:
May be we can ignore errors here. If there's any error, it will use default values.

Reply:
Fine